org.eclipse.m2e.profiles.core/src/org/eclipse/m2e/profiles/core/internal/management/ProfileManager.java

```diff
@@ -64,15 +66,23 @@
  * @author Fred Bricon
  * @since 1.5.0
  */
+@Component(service = IProfileManager.class)
 public class ProfileManager implements IProfileManager {
 
+  @Reference
+  IProjectConfigurationManager configurationManager;
+
+  @Reference
+  IMaven maven;
+
+  @Reference
+  ILog log;
```

Comment:
This does not work because the platform does not register an ILog service implementation.

Reply:
It will on the next release... so one need an ibuild currently.

Reply:
Understand, great.
For curiosity, how do you ensure that a component gets the Log associated with its bundle injected?
Can you please link the PR?

Reply:
See 
- https://github.com/eclipse-platform/eclipse.platform.runtime/pull/34

Reply:
Thanks. Learned one more thing.